src/emulator/auth/widget_ui.ts

```diff
@@ -294,21 +327,67 @@ const STYLE = `
   color: #858585;
 }
 
-body {
-  font-family: "Roboto", sans-serif;
-  margin: 0;
-  padding: 0;
-  width: 100%;
+.subtitle {
+ color: var(--mdc-theme-text-secondary-on-background);
+ font-size: 14px;
+ line-height: 20px;
+ margin-block-end: 0em;
+ margin-block-start: 0em;
 }
 
 #content {
   box-sizing: border-box;
-  padding: 12px;
-  width: 500px;
+  width: 515px;
+  margin: 16px auto;
 }
 
-button {
-  text-transform: none !important;
+.content-wrapper {
+  padding: 0 24px;
+}
+
+.mdc-list--avatar-list .mdc-list-item {
+  padding: 0 24px;
+}
+
+.mdc-list--avatar-list .mdc-list-item__graphic {
+  border-radius: 50%;
+  height: 36px;
+  margin-left: 0;
+  margin-right: 16px;
+  width: 36px;
+}
+
+.mdc-list .mdc-list-item__graphic {
+  display: -ms-inline-flexbox;
```

Comment:
I don't know if we really need all this -ms prefixed stuff, do we even support ie? (I don't think we do in the console)

Reply:
Ah, copied some of this from material website. I'll drop it.